packages/bot-web-ui/src/components/transaction.jsx

```diff
@@ -130,41 +134,62 @@ const Transaction = ({ active_transaction_id, contract, setActiveTransactionId }
         message={contract && <PopoverContent contract={contract} />}
     >
         <div
-            className='transactions__item'
+            className={classNames('transactions__item', {
+                'transactions__item-mobile': is_mobile,
```

Comment:
Try to use correct BEM naming, i.e. `mobile` changes the appearance of the element, so `--mobile` would be more correct.